mofacts/client/views/login/signInSouthwest.js

```diff
@@ -118,6 +118,7 @@ setClass = function(curClassID) {
   const allClasses = Session.get('curTeacherClasses');
   const curClass = allClasses.find((aClass) => aClass.sectionid == curClassID);
   Session.set('curClass', curClass);
+  Session.set('curSectionId', curClass.sectionid)
```

Comment:
This session variable should be cleared when the user logs out with the logout index button